src/lib/api/builder.ts

```diff
@@ -314,35 +318,35 @@ function toRemoteCollection(collection: Collection): RemoteCollection {
     managers: collection.managers,
     contract_address: collection.contractAddress || null,
     salt: collection.salt || null,
-    items: [],
     created_at: new Date(collection.createdAt),
     updated_at: new Date(collection.updatedAt)
   }
 
   return remoteCollection
 }
 
-function fromRemoteCollection(remoteCollection: RemoteCollection): CollectionWithItems {
-  const collection: CollectionWithItems = {
+function fromRemoteCollection(remoteCollection: RemoteCollection) {
+  const collection: Collection = {
     id: remoteCollection.id,
     name: remoteCollection.name,
     owner: remoteCollection.eth_address,
     isPublished: remoteCollection.is_published,
     isApproved: remoteCollection.is_approved,
     minters: remoteCollection.minters || [],
     managers: remoteCollection.managers || [],
-    items: [],
     createdAt: +new Date(remoteCollection.created_at),
     updatedAt: +new Date(remoteCollection.updated_at)
   }
 
+  let items: Item[] = []
+
   if (remoteCollection.salt) collection.salt = remoteCollection.salt
   if (remoteCollection.contract_address) collection.contractAddress = remoteCollection.contract_address
   if (remoteCollection.items) {
-    collection.items = remoteCollection.items.map(item => fromRemoteItem(utils.omit(item, ['collection'])))
+    items = remoteCollection.items.map(item => fromRemoteItem(utils.omit(item, ['collection']))[0])
```

Comment:
I know the rest of the app is using arrays to return multiple values but I was wondering, why not just returning an object?
```diff
- [item, collection]
+ { item, collection }
```

that'd allow for doing this: `fromRemoteItem(utils.omit(item, ['collection'])).item` instead of having to figure out what `[0]` is in that context

I also ask because it's not really obvious that `fromRemoteItem` returns two values